tgui/packages/tgui/interfaces/common/stringUtils.ts

```diff
@@ -1,3 +1,15 @@
+import { randInt } from './mathUtils';
+
 export const pluralize = (word: string, n: number) => (n !== 1 ? word + 's' : word);
 
 export const capitalize = (word: string) => word.replace(/(^\w{1})|(\s+\w{1})/g, letter => letter.toUpperCase());
+
+const glitches = ['$', '{', ']', '%', '^', '?', '>', '¬', 'π', ';', 'и', 'ю', '/', '#', '~'];
+export const glitch = (text: string, amount: number) => {
+  const chars = text.split('');
+  for (let i = 0; i < amount; i++) {
+    const charIndex = randInt(0, chars.length);
```

Comment:
```suggestion
    const charIndex = randInt(0, chars.length - 1);
```
I think this might be a bug in the initial implementation.